src/legacy/templates/chartEditBarlineExtras.handlebars

```diff
@@ -25,9 +25,10 @@
         <option value="ShortDash">ShortDash</option>
         <option value="ShortDot">ShortDot</option>
         <option value="ShortDashDot">ShortDashDot</option>
-        <option value="ShortDashDotDot">ShortDashDotDot</option>
+        {{!-- Hidden from view since they do not display well --}}
+        {{!-- <option value="ShortDashDotDot">ShortDashDotDot</option>
         <option value="LongDashDot">LongDashDot</option>
-        <option value="LongDashDotDot">LongDashDotDot</option>
+        <option value="LongDashDotDot">LongDashDotDot</option> --}}
```

Comment:
Just a minor formatting thing, but I think it would be clearer if individual lines were commented rather than multiple lines as a block. Ie. it's not clear initially (via git blame or similar) that `LongDashDot` is disabled unless you look at the lines above and below it.